server/generated.ts

```diff
@@ -51,14 +51,14 @@ export enum Visibility {
 /** Metadata about a room, according to the Monolith. */
 export interface RoomMetadata {
 	name: RoomName;
-	title: string;
-	description: string;
-	isTemporary: boolean;
+	title: unknown;
+	description: unknown;
+	isTemporary: unknown;
```

Comment:
Did you manually change these to unknown? Typeshare doesn't treat these as unknown by default, and it doesn't have an escape hatch to set the type manually. The codegen script has a hack to manually tweak the output of typeshare so it works a little better for us (that's why `currentSource` wasn't causing problems)

Reply:
Yes, I ran the typeshare script and these didn't get updated (or well, they did but to `Value`)

Reply:
The CI job enforces that running the codegen script produces the exact same output as what is in `generated.ts`